staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles/options.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profiles
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type DebugPodOptions struct {
```

Comment:
Can we not just pass `DebugOptions` to the profiles?

Reply:
I think it would cause cyclic import if we make the debugger rely on the `DebugOptions` and I think it might be more elegant to pass the source pod via `DebugPodOptions` than adding an argument to the method `NewPod`.